python/pyspark/ml/tests.py

```diff
@@ -590,10 +590,14 @@ def test_fit_minimize_metric(self):
         tvsModel = tvs.fit(dataset)
         bestModel = tvsModel.bestModel
         bestModelMetric = evaluator.evaluate(bestModel.transform(dataset))
+        validationMetrics = tvsModel.validationMetrics
 
         self.assertEqual(0.0, bestModel.getOrDefault('inducedError'),
                          "Best model should have zero induced error")
         self.assertEqual(0.0, bestModelMetric, "Best model has RMSE of 0")
+        self.assertEqual(len(grid), len(validationMetrics),
+                         "validationMetrics has the same size of grid parameter")
+        self.assertIn(bestModelMetric, validationMetrics)
```

Comment:
`self.assertEqual(bestModelMetric, max(validationMetrics))`? It's more explicit IMO.